performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -17,20 +17,77 @@
 
 #include <sys/time.h>
 #include <sys/resource.h>
-
+#include <boost/uuid/uuid_io.hpp>
+#include <cstddef>
 #include <chrono>
 #include <sstream>
 #include <string>
 
 #include "../utilities/statistics_tracker.hpp"
+#ifdef ODB_FOR_SQL_ENABLED
+  #include <odb/core.hxx>
+  #include "experiment_configuration.hpp"
+#endif
 
 namespace performance_test
 {
 
 /// Outstream operator for timeval to seconds (double).
 std::ostream & operator<<(std::ostream & stream, const timeval & e);
+#ifdef ODB_FOR_SQL_ENABLED
+class RusageTracker
+{
+public:
+  void setValues(rusage sys_usage) const
```

Comment:
Where do we call this function?

Reply:
In analysis_result.cpp 

m_sys_tracker.setValues(m_sys_usage);

Reply:
Okay great